hibernate-testing/hibernate-testing.gradle

```diff
@@ -16,6 +16,8 @@ dependencies {
     api testLibs.junit4
     api testLibs.junit5Api
     api testLibs.junit5Params
+    api testLibs.junit5Engine
```

Comment:
Minor, but can't these be `implementation`?

Reply:
we don't have so many choices in Maven 😃 so I just placed it next to other JUnit libs, but yes, it definitely can go to `implementation` 👍🏻 thanks!